h2o-core/src/test/java/water/UnknownHeartbeatTest.java

```diff
@@ -14,6 +14,7 @@
 
   @Test
   public void testIgnoreUnknownHeartBeat() {
+    final int clientsCountBefore = H2O.getClients().size();
```

Comment:
Looks good. Just explanation why this change is fine:

We are just interested if a new client was recorded by the current node. Since the H2O nodes are shared between tests it's not the safest to test again specific number, but we gan get previous number of client and test against that